docs/checks/internal/reader.go

```diff
@@ -0,0 +1,53 @@
+// Copyright 2021 Security Scorecard Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package idocs contains internal functions for reading input YAML file.
+package idocs
```

Comment:
Nit:`idocs` sounds like an Apple product to me :) Maybe `internaldocs`?

Reply:
hahah. Will update.